backend/battles/tests/test_views.py

```diff
@@ -252,3 +252,44 @@ def test_opponent_select_pokemons(self, send_mock):
 
         send_mock.assert_called_with(battle)
         self.assertEqual(response.status_code, 302)
+
+
+class SettledBattlesListViewTests(TestCase):
+    def setUp(self):
+        self.client = Client()
+
+    def test_list_settled_battles(self):
+        creator = mommy.make("users.User")
+        opponent = mommy.make("users.User")
+
+        pokemon_1 = mommy.make("pokemons.Pokemon")
+        pokemon_2 = mommy.make("pokemons.Pokemon")
+        pokemon_3 = mommy.make("pokemons.Pokemon")
+
+        battle_1 = mommy.make(
+            "battles.Battle",
+            creator=creator,
+            opponent=opponent,
+            creator_pokemon_1=pokemon_1,
+            creator_pokemon_2=pokemon_2,
+            creator_pokemon_3=pokemon_3,
+        )
+
+        battle_2 = mommy.make(
+            "battles.Battle",
+            creator=creator,
+            opponent=opponent,
+            creator_pokemon_1=pokemon_1,
+            creator_pokemon_2=pokemon_2,
+            creator_pokemon_3=pokemon_3,
+            opponent_pokemon_1=pokemon_3,
+            opponent_pokemon_2=pokemon_2,
+            opponent_pokemon_3=pokemon_1,
+        )
+
+        url = reverse("battles:settled_battles_list")
+        response = self.client.get(url)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertNotContains(response, f"Pokebattle {battle_1.id}")
```

Comment:
We normally try to avoid checking for text in the response when we are working on the backend. Instead, we normally check for the things passed to the `response.context`